bemani/backend/iidx/rootage.py

```diff
@@ -0,0 +1,1299 @@
+            qpro_dict.replace_int('body', int(qpro_equip.attribute('body')))
+            newprofile.replace_dict('qpro', qpro_dict)
+
+        # Qpro secret unlocks in step-up mode
+        qpro_secret = request.child('qpro_secret')
+        if qpro_secret is not None:
+            qpro_secret_dict = newprofile.get_dict('qpro_secret')
+            qpro_secret_dict.replace_int_array('head', 5, qpro_secret.child_value('head'))
+            qpro_secret_dict.replace_int_array('hair', 5, qpro_secret.child_value('hair'))
+            qpro_secret_dict.replace_int_array('face', 5, qpro_secret.child_value('face'))
+            qpro_secret_dict.replace_int_array('body', 5, qpro_secret.child_value('body'))
+            qpro_secret_dict.replace_int_array('hand', 5, qpro_secret.child_value('hand'))
+            newprofile.replace_dict('qpro_secret', qpro_secret_dict)
+
+        # Orb data saving
+        orb_data = request.child('orb_data')
+        if orb_data is not None:
+            orbs = newprofile.get_int('orbs')
+            orbs = orbs + int(orb_data.attribute('add_orb'))
+            if orb_data.child_value('use_vip_pass'):
+                orbs = 0
+            newprofile.replace_int('orbs', orbs)
+            # present_orb?
+
+        # Keep track of play statistics across all mixes
+        self.update_play_statistics(userid, play_stats)
+
+        # dj_rank?
+        # pay_money_data?
+        # music_history?
+        # play_log?
+
+        return newprofile
+
+
+    # 0x1803DF520
+    def handle_IIDX26pc_request(self, request: Node) -> Node:
+        method = request.attribute('method')
+        root = Node.void('IIDX26pc')
+
+        # 0x18067BC70
+        if method == 'get':
+            # from sinobuz.py
+            refid = request.attribute('rid')
+            profile = self.get_profile_by_refid(refid)
+
+            if profile is not None:
+                root = profile
+        
+        # 0x18067BCA0
+        if method == 'reg':
+            # from sinobuz.py
+            refid = request.attribute('rid')
+            name = request.attribute('name')
+            pid = int(request.attribute('pid'))
+            profile = self.new_profile_by_refid(refid, name, pid)
+
+            if profile is not None:
+                root.set_attribute('id', str(profile.get_int('extid')))
+                root.set_attribute('id_str', ID.format_extid(profile.get_int('extid')))
+        
+        # 0x18067BD90
+        if method == 'common':
+            # expire
+            root.set_attribute('expire', '600')
+
+            # fixme: monthly_mranking, total_mranking
+            # todo: internet ranking (i cannot find any infomation about this)
+            # todo: cm, what is it
+
+            # KONAMI Arcade Championship 8th
+            root.add_child(Node.string('kac_mid', '26033'))
+            root.add_child(Node.string('kac_clid', str(self.CHART_TYPE_A7))) # not sure :/

```

Comment:
You can usually get this information from bemaniwiki 2nd.

Reply:
Still can't find the chart information🤔